quickstart/java-redisson/redis-redisson-test/src/main/java/example/demo/App.java

```diff
@@ -0,0 +1,39 @@
+package example.demo;
+
+import org.redisson.Redisson;
+import org.redisson.api.RMap;
+import org.redisson.api.RedissonClient;
+import org.redisson.config.Config;
+
+import java.time.LocalDateTime;
+
+/**
+ * Redis test
+ *
+ */
+public class App {
+    public static void main(String[] args) {
+        // Connect to the Azure Cache for Redis over the TLS/SSL port using the key
+        Config config = new Config();
+        config.useSingleServer().setPassword(System.getenv("REDISCACHEKEY"))
+                .setAddress(String.format("rediss://%s:6380", System.getenv("REDISCACHEHOSTNAME")));
+        RedissonClient client = Redisson.create(config);
+
+        // Perform cache operations using the cache client
+        RMap<String, String> cache = client.getMap("namedCache");
```

Comment:
Since I'm not too familiar with the Redisson library, I'm wondering if the names `cache` and `namedCache` are good matches for the Redisson concept of a map.

From a quick skim of the Redisson docs, my first impression is that they tend to use `map` as the variable name, and the string they pass to `getMap()` doesn't typically include the word "cache" either:

```
RMap<String, String> map = redisson.getMap("test");
```

I'm OK with either the current naming in this PR or naming similar to the Redisson samples, depending on what fits best with how Redisson works. Based on <https://github.com/redisson/redisson/wiki/11.-Redis-commands-mapping>, it looks like `RMap` is specifically for operations where Redis automatically does hashing. It looks like this is the most popular way to use a Redis cache from Redisson, but Redisson technically does support other methods of setting values. It looks like the convention of preferring RMap in Redisson might be inspired by _Use hashes when possible_ as mentioned on <https://redis.io/topics/memory-optimization>.

Long story short, I think maybe borrowing the `map` naming from the Redisson samples will be a good fit?

Reply:
Good catch again! Updated the variable naming in commit 5e6be51.